akka-persistence/src/test/scala/akka/persistence/PersistentActorSpec.scala

```diff
@@ -1132,6 +1177,49 @@ abstract class PersistentActorSpec(config: Config) extends PersistenceSpec(confi
       expectMsg(List("b-1", "b-2"))
     }
 
+    "be extendible by hooking into onPersisted, which is called after persist" in {
+      val persistentActor = namedPersistentActor[PublishAfterPersistActor]
+      val listener = TestProbe()
+      system.eventStream.subscribe(listener.ref, classOf[String])
+      persistentActor ! "a"
+      expectMsg(982417)
+      listener.expectMsg("message: 982417")
+    }
+    "not call the callback hook when onPersistSuccess throws an exception" in {
+      val persistentActor = namedPersistentActor[ExceptionInHookAfterPersistActor]
+      val listener = TestProbe()
+      system.eventStream.subscribe(listener.ref, classOf[String])
+      persistentActor ! "a"
+      expectNoMsg(100.millis)
```

Comment:
This line can also be dropped: for a negative assertion it is too short, but extraneous messages will be detected in following tests anyway.